### ISSUE TITLE
Application depicts black screen within dialog on first install. Workaround posted.

### ISSUE THREAD

**Issue by NOVATechnocrat** [cerebroapp/cerebro#531]
Installed it, typed ctrl + space, and the application presented itself as expected. When I started typing in the query box, the dialog expanded and presented a black screen only, with what looked like a Google icon. Clearly not as expected.

After looking around I saw that cerebro had theme settings and since I use a dark theme, many things commonly have issues supporting it so I tried to change theme setting by hitting ctrl + space and then typing "cerebro settings" to access settings. I then say artifacts in the black screen. I tried clicking all around the black background and eventually clicked on the theme selection box when the dialog normalized as expected and I was able to use the app.

In short, for anyone with this problem or something similar:

Hit Ctrl + space, type cerebro settings, and then keep clicking around in the black dialog as the menu elements are there, just not depicted properly. 

Pop OS 20.10 and installed 4.0 of cerebro as instructed in the source readme:

Install 
First, clone the repo via git: 
$ git clone https://github.com/KELiON/cerebro.git cerebro

And then install dependencies: 
$ cd cerebro && yarn && cd ./app && yarn && cd ../

Run 
$ yarn run dev

**Comment by GabrielGCardoso** [comment]
Same here on ubuntu 20.04
but if change the global theme i can use normally
- `app/main/css/global.css`
- change line 9 from `background-color: var(--main-background-color);` to `background-color: #fff;` 
![Screenshot from 2021-05-25 19-46-28](https://user-images.githubusercontent.com/21351737/119578130-5a223c00-bd92-11eb-8f16-fa07b62be952.png)
![Screenshot from 2021-05-25 19-46-38](https://user-images.githubusercontent.com/21351737/119578132-5abad280-bd92-11eb-86c1-ccc97dffa092.png)
![Screenshot from 2021-05-25 19-46-53](https://user-images.githubusercontent.com/21351737/119578133-5abad280-bd92-11eb-9030-b5e42bf1fba7.png)
![Screenshot from 2021-05-25 19-47-05](https://user-images.githubusercontent.com/21351737/119578134-5b536900-bd92-11eb-8d1b-45a889e6bbbb.png)

**Comment by GabrielGCardoso** [comment]
I believe that this issue could be closed when the program start automatically defining a default theme (black or white) in ~ / .config / Cerebro / config.json

**Comment by oguhpereira** [comment]
we have a problem setting our default theme today.

This problem can be quickly resolved by removing the .config/ Cerebro/ folder and generating another one.

But we need help to remove this bug once and for all
